workflows/common/swift/obj_app.swift

```diff
@@ -9,11 +9,13 @@
 */
 (string obj_result) obj(string params,
                         string expid,
-                        string runid)
+                        string runid,
+                        string model_name,
+                        string candle_image)
```

Comment:
Again, this function signature change would break other workflows @j-woz what do you think is the best work around? 

How about another obj for singularity say '`obj_singularity`'?

Reply:
Now there is a new obj, obj_container added to the workflows/common/swift. obj_app.swift is not changed.